<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R3EKG9O" ignored="1" name="Personal Loan" tests="2" failures="1" timestamp="2020-06-21T17:49:27 GMT" time="0.051" errors="0">
  <testcase name="Demo" time="0.01" classname="firsttestngpackage.day1"/>
  <testcase name="ploan" time="0.01" classname="firsttestngpackage.day2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firsttestngpackage.day2.ploan(day2.java:12)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- ploan -->
  <testcase name="SecondTest" time="0.0" classname="firsttestngpackage.day1">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal Loan -->
